src/code/z_player_call.c

```diff
@@ -1,6 +1,6 @@
 #include "global.h"
 
-#define FLAGS 0x06000035
+#define FLAGS ACTOR_FLAG_0 | ACTOR_FLAG_2 | ACTOR_FLAG_4 | ACTOR_FLAG_5 | ACTOR_FLAG_25 | ACTOR_FLAG_26
```

Comment:
The content of these defines should be wrapped in parens imo.
Also on the topic of these defines, I think they can avoid cluttering the init vars so they are useful to some extent, but they should probably be called `INIT_FLAGS` since they are specifically the flags on init

Reply:
i think i will probably repurpose my THIS script to get rid of this define just to see what it looks like, and we can decide from there

Reply:
it will just look like long lines in the init vars, which is probably fine tbh

should these changes come in the scope of the PR? I'm fine with it, just need to decide what we actually do

Reply:
imo no, lets just get this in and see how it looks later